pkg/joinserver/http_interop.go

```diff
@@ -119,26 +124,40 @@ func (srv interopServer) JoinRequest(ctx context.Context, in *interop.JoinReq) (
 func (srv interopServer) HomeNSRequest(ctx context.Context, in *interop.HomeNSReq) (*interop.HomeNSAns, error) {
 	ctx = log.NewContextWithField(ctx, "namespace", "joinserver/interop")
 
-	netID, err := srv.JS.GetHomeNetID(ctx, types.EUI64(in.ReceiverID), types.EUI64(in.DevEUI), X509DNAuthorizer)
+	netID, nsID, err := srv.JS.GetHomeNetID(ctx, types.EUI64(in.ReceiverID), types.EUI64(in.DevEUI), InteropAuthorizer)
 	if err != nil {
+		switch {
+		case errors.Resemble(err, errRegistryOperation):
+			if errors.IsNotFound(errors.Cause(err)) {
+				return nil, interop.ErrUnknownDevEUI.WithCause(err)
```

Comment:
Could also be `ReceiverID` / `JoinEUI` - maybe unknown EUIs ?

Reply:
Yes, in The Things Stack only the combination of JoinEUI and DevEUI is supposed to be unique. Generally speaking, LoRaWAN Network Servers keep DevEUIs unique, regardless of the JoinEUI.

Therefore there is no error code in Backend Interfaces that says `UnknownEUIs`; it's either `UnkonwnDevEUI` or `UnknownReceiver`.

Now, `UnkownReceiver` (sic) should be returned when the `ReceiverID` isn't known, i.e. when you're contacting a Join Server that is not handling this JoinEUI. We might want to support this through config (like a list of JoinEUI prefixes), but I don't really see a need for it.